backend/data/clients/firebase/write.go

```diff
@@ -2,17 +2,24 @@ package firebase
 
 import (
 	"context"
+	"errors"
 
 	log "github.com/sirupsen/logrus"
 )
 
 func (u *User) Write() error {
 	// todo: consider logging ref or actual document info here
 	log.Infof("writing user to firestore: %+v", u)
-
-	ref, _, err := Client.Collection("users").Add(context.Background(), u)
-	u.FirebaseID = ref.ID
-
+	// check if user with provided email already exists
```

Comment:
For now, for ease of development I say that we just use Email. Setting it to the UID will require tokens to be sent. We can implement that in phase 2 after we have basic functionality working better